test/unit/BPool/BPool_Bind.t.sol

```diff
@@ -11,15 +11,15 @@ contract BPoolBind is BPoolBase {
     bPool.call__setLock(_MUTEX_TAKEN);
     vm.expectRevert(IBPool.BPool_Reentrancy.selector);
     // it should revert
-    bPool.bind(token, tokenBindBalance, tokenWeight);
+    bPool.bind(tokens[0], tokenBindBalance, tokenWeight);
```

Comment:
hmmm i'm not sure about Bind particularly, since `token` makes more sense here than `tokens[0]` (since the token shouldn't be already binded), but also don't have strong opinions about it, nit

Reply:
referring to the token like this allows us to get rid of the state variables for each one so I prefer it